.github/scripts/collect-deb-package.sh

```diff
@@ -18,6 +18,20 @@ AUTHOR_EMAIL="contact@centreon.com"
 # fix version to debian format accept
 VERSION="$(echo $VERSION | sed 's/-/./g')"
 
+echo "[DEBUG] - VERSION = $VERSION"
+echo "[DEBUG] - VERSION RELEASE = $VERSION-$RELEASE"
+
+MAJOR_LEFT_PART=$( echo $VERSION | cut -d . -f 1 )
+MAJOR_RIGHT_PART=$( echo $VERSION | cut -d - -f 1 | cut -d . -f 2 )
+BUMP_MAJOR_RIGHT_PART=$(( MAJOR_RIGHT_PART+1 ))
+if [ ${#BUMP_MAJOR_RIGHT_PART} -eq 1 ]; then
+    # Add a zero before the new single numeric char
+    BUMP_MAJOR_RIGHT_PART="0$BUMP_MAJOR_RIGHT_PART"
+fi
+MAJOR_THRESHOLD="$MAJOR_LEFT_PART.$BUMP_MAJOR_RIGHT_PART"
+
+echo "[DEBUG] - VERSION = $MAJOR_THRESHOLD"
```

Comment:
don't you want to write `"[DEBUG] - VERSION THRESHOLD = $MAJOR_THRESHOLD"` ?
Otherwise, you'll get two lines containing `VERSION = ` with different values, a little bit weird.

Reply:
indeed. But this will not be kept. The debug part will be removed

Reply:
This part has been removed